readthedocs/privacy/backends/syncers.py

```diff
@@ -8,12 +14,17 @@
 log = logging.getLogger(__name__)
 
 
+# The file builting was removed in Python 3, so we should not use it
+# Setting to None here to silence pylint for this builtin only
+file = None  # pylint: disable=redefined-builtin
```

Comment:
That was my first thought, but I wasn't sure if my searches were finding everything. I'll go ahead and do this.